cardano-node/src/Cardano/Wallet/Client.hs

```diff
@@ -112,22 +112,23 @@ localInitiatorNetworkApplication Proxy chainSyncTracer localTxSubmissionTracer c
       (NodeToClientVersionData { networkMagic = nodeNetworkMagic (Proxy @blk) cfg })
       (DictVersion nodeToClientCodecCBORTerm) $ \_peerid ->
 
-    nodeToClientProtocols
-      (InitiatorProtocolOnly $
-         MuxPeerRaw $ \channel -> do
-           txv <- newEmptyTMVarM @_ @(GenTx blk)
-           runPeer
-             localTxSubmissionTracer
-             localTxSubmissionCodec
-             channel
-             (localTxSubmissionClientPeer
-                 (txSubmissionClient @(GenTx blk) txv)))
-      (InitiatorProtocolOnly $
-         MuxPeer
-           chainSyncTracer
-           (localChainSyncCodec @blk cfg)
-           (chainSyncClientPeer chainSyncClient))
-
+    nodeToClientProtocols $
+      NodeToClientProtocols
+        { localChainSyncProtocol =(InitiatorProtocolOnly $
+           MuxPeerRaw $ \channel -> do
+             txv <- newEmptyTMVarM @_ @(GenTx blk)
+             runPeer
+               localTxSubmissionTracer
+               localTxSubmissionCodec
+               channel
+               (localTxSubmissionClientPeer
+                   (txSubmissionClient @(GenTx blk) txv)))
+        , localTxSubmissionProtocol =(InitiatorProtocolOnly $
+           MuxPeer
+             chainSyncTracer
+             (localChainSyncCodec @blk cfg)
+             (chainSyncClientPeer chainSyncClient))
+        }
```

Comment:
These are the wrong way around.

What is this being used for? It is clearly not being tested.

Reply:
The wallet was removed from the repo so I need to rebase on master.